src/main/java/com/jcraft/jsch/JSch.java

```diff
@@ -38,7 +38,7 @@
   /**
    * The version number.
    */
-  public static final String VERSION  = Version.getVersion();
+  public static final String VERSION  = "0.2.0"; //Version.getVersion();
```

Comment:
This part needs to be reverted.

Reply:
That part wasn't intended to be in the commit